### ISSUE TITLE
 NBench.Runner execution failed when the target assembly is signed

### ISSUE THREAD

**Issue by AlejandroPiCano** [petabridge/NBench#83]
Hi, 

I'm trying to run the NBench.Runner for an assembly signed. I have generated the signature for NBench.dll and NBench.Runner.exe, with the same signature of my project using ildasm and ilasm tools as they say in http://buffered.io/posts/net-fu-signing-an-unsigned-assembly-without-delay-signing/. When run NBench.Runner.exe from my command prompt see this error:

**System.IO.FileLoadException: You can not load file or assembly 'NBench, Version=0.1.6.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. An assembly with strong name is required. (Exception of HRESULT: 0x80131044) in NBench.Runner.Program.Main(String[] args).**

However all references to NBench.dll and NBench.Runner.exe are signed. I do not know why the error occurs. Please help me with this issue.

Thanks.

**Comment by Aaronontheweb** [comment]
@AlejandroPiCano hmmm.... I'm not sure offhand either... Do you have any dependencies that might still be unsigned? cc @twenzel - any chance there could be an `AppDomain` issue at work here?

**Comment by AlejandroPiCano** [comment]
Hi @Aaronontheweb and @twenzel ,

I thought at first that this could be a problem, that the rest of the despencies were not signed, but my project has signed all dependencies, then to test this problem did the following:
- I created a completely new project with the Github example class CounterPerfSpecs.
- I added references to NBench.
- I created a new key "mykey.snk"
- I signed my project.
- I signed the references to NBench of ... \ bin \ Debug and ...packages \ NBench.Runner.0.1.6 \ lib \ net45.
- I recompile the solution.
- I run NBench.Runner.

The results were the same, I got the same error. I think if i can solve the problem for this simple example, i can apply the solution to my project and run NBench.Runner succesfully.

 I have attached the message error: 
![nbench runner error](https://cloud.githubusercontent.com/assets/7825407/13723296/42a2663e-e860-11e5-96f2-5f84c51b5910.PNG)

Sorry for my english, thanks.
